ethicalads-theme/templates/ea/publishers.html

```diff
@@ -24,13 +24,13 @@
       <div class="col-12 col-md-6">
 
         <!-- Heading -->
-        <h1 class="display-3 font-weight-light" data-aos="fade-left">Developers: make money with your content while respecting user privacy</h1>
+        <h1 class="display-3 font-weight-light" data-aos="fade-left">Developers: join the ad network with no tracking</h1>
```

Comment:
```suggestion
        <h1 class="display-3 font-weight-light" data-aos="fade-left">Developers: join the ad network without tracking</h1>
```

This is just a thought